### DISCUSSION TITLE
Download .csv with semi-colon ";" separator ?

### DISCUSSION THREAD

**Discussion by ThierryM** [sqlpage/SQLPage#345] [Q&A] [answered]
Hi,
I would download data from SQLpage with ";" separator but I don' succeed. The .csv file obtained has the "," as separator.
My code to export is :
`SELECT 
    'csv' as component,
    'Exporter le fichier des établissements ' as title,
    'etablissements' as filename,
    'file-download' as icon,
    'green' as color,
    ';' as separator;`
Where is my mistake ?
Thanks for your help.

**Comment by plabuse** [comment]
idem for me with sqlpage v0.21.0
I like sqlpage

**Comment by lovasoa** [comment, marked as answer]
Thanks for reporting this ! I just fixed it, it will be in the next version.

**Comment by ThierryM** [comment]
Hi,
I've tried the .csv export with separator ";" with the last version of SQLPage (main docker image 2024-06-07) but the .csv file generated has still the "," as separator.
Regards

**Reply by lovasoa** [reply]
Are you sure you are using the latest version ?

The official website documentation is up to date with the latest version, and the separator property works there:

![image](https://github.com/lovasoa/SQLpage/assets/552629/04d3f8c4-c5c0-4677-8564-9d5a9aae9715)

**Comment by ThierryM** [comment]
Yes, I use the last Docker Image (downloaded this morning). I have the tootips on the buttons working.
It's working on the official website documentation, I confirm.

OH SORRY !!! I used ";" in my code in place of ';'
Sorry for the noise.
Regards.

**Comment by plabuse** [comment]
Hello, i use sqlpage with linux binary 0.22.0 and for me the issue is not done.
![Capture du 2024-06-08 08-09-42](https://github.com/lovasoa/SQLpage/assets/12814628/d01838ab-c7e6-416e-a7c1-894b2f21890e)
French : j'ai surement oublié un truc.

**Reply by lovasoa** [reply]
Le problème est réglé dans la prochaine version, qui n'est pas encore sortie officiellement, mais que l'on peut déjà tester avec `docker pull lovasoa/sqlpage:main`

**Reply by plabuse** [reply]
Ok, thanks for the work.
